### PR TITLE
Minor README.md improvements

### PR DESCRIPTION
**Pull request by bastelfreak**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,15 +28,31 @@ configuration layout changed significantly.
...
 
 * Fork the project
 * Commit and push until you are happy with your contribution
 * Send a pull request with a description of your changes
 
-# More info
+## More info
 
-See http://theforeman.org or at #theforeman irc channel on freenode
+See <http://theforeman.org> or at #theforeman irc channel on freenode
```
Comment (ekohl) [inline README.md]: While you're at it, maybe change this to https?

File: README.md [hunk 2]
```diff
@@ -3,7 +3,7 @@
 Installs and configures the Foreman Smart Proxy and services that it can
 interact with, e.g. DNS, DHCP and Puppet and TFTP.
 
-Part of the Foreman installer: http://github.com/theforeman/foreman-installer
+Part of the Foreman installer: <http://github.com/theforeman/foreman-installer>
```
Comment (ekohl) [inline README.md]: While you're at it, maybe change this to https?

**Comment by bastelfreak** [comment]
@ekohl fixed

**Comment by mmoll** [comment]
merged, danke @bastelfreak!

**Pull request merged**